packages/manager/apps/web/client/app/domain/zone/product-zone-activate/activate.service.js

```diff
@@ -0,0 +1,69 @@
+import find from 'lodash/find';
+
+export default class DomainDnsZoneActivateService {
+  /* @ngInject */
+  constructor(WucOrderCartService) {
+    this.WucOrderCartService = WucOrderCartService;
+  }
+
+  async getServiceOption(serviceName) {
```

Comment:
Imo we should avoid using `async` in services as it will forces us to `$timeout` everything in controller